pkg/autodiscovery/integration/config.go

```diff
@@ -78,7 +78,7 @@ type Config struct {
 
 	// ServiceID is the ID of the service (set only for resolved templates and
 	// for service configs)
-	ServiceID string `json:"-"` // (include in digest: true)
+	ServiceID string `json:"service_id"` // (include in digest: true)
```

Comment:
Does the comment need updating for this field?

Reply:
No, it's included in the digest and this PR doesn't change that.

Reply:
It still applies. Do you think it's worth it to add a comment here that says that values provided explicitly by users are ignored?